gtkm/auth/service_auth.py

```diff
@@ -1,13 +1,67 @@
-from fastapi import APIRouter
+from fastapi import APIRouter, Depends
+from fastapi.encoders import jsonable_encoder
+from fastapi.responses import RedirectResponse, JSONResponse
+from sqlalchemy.orm import Session
+import requests
+import uuid
 
+import crud, model, schemas
+from database import engine, get_db
+
+model.Base.metadata.create_all(bind=engine)
 auth = APIRouter()
 
+root_url = "http://127.0.0.1:8000"
+client_id = "<CLIENT_ID>"
+client_secret = "<CLIENT_SECRET>"
+current_user_id = None
+
 
 @auth.get("/auth/user/")
-def get_user():
-    return {"id": 111, "github": "zygfryd", "gitlab": None}
+def get_user(id: str, db: Session = Depends(get_db)):
+    user = crud.get_user(db, id)
+    if user:
+        user_json = jsonable_encoder(user)
+        return JSONResponse(content=user_json)
+    else:
+        return "No user with id: " + id + " exists in the database."
```

Comment:
Proponuję zamianę na:

```python
raise HTTPException(status_code=status.HTTP_404_NOT_FOUND, detail="User ID does not exist.")
```

Proszę również o analogiczną zmianę kodów odpowiedzi HTTP dla pozostałych endpointów.

Więcej informacji znajdziesz w moim [komentarzu](https://github.com/mpfmorawski/git-to-know-me/issues/7#issuecomment-1107453067) pod Issue #7.

Reply:
Done